sdks/java/io/amazon-web-services/build.gradle

```diff
@@ -42,4 +42,5 @@ dependencies {
   shadowTest library.java.mockito_core
   shadowTest library.java.junit
   shadowTest library.java.slf4j_jdk14
+  testCompile "io.findify:s3mock_2.12:0.2.4"
```

Comment:
I had not seen that s3mock required scala, can you please use version 2.11 (just for uniformity with the rest of the project).

Reply:
Done